rest-api/profile-info/middlewares/validateProfileDetails.js

```diff
@@ -0,0 +1,22 @@
+function validateProfileDetails(req, res, next) {
+    const { profile_pic, first_name, last_name, bio } = req.body
```

Comment:
I believe this will fail if I don't supply one of those fields, you should check if they exist before deconstructing body and defining them